packages/babel-preset-env/src/normalize-options.js

```diff
@@ -15,6 +15,48 @@ const validIncludesAndExcludes = new Set([
   ...defaultWebIncludes,
 ]);
 
+const validRegExp = (regexp: string): boolean => {
+  try {
+    new RegExp(regexp);
+    return true;
+  } catch (e) {
+    return false;
+  }
+};
+
+const selectPlugins = (regexp: RegExp): Array<string> =>
+  Array.from(validIncludesAndExcludes).filter(item => item.match(regexp));
```

Comment:
Notice that with `new RegExp(value)`, for case like `es6.reflect.set`, it will also match the `es6.reflect.set-prototype-of`, which is a different built-in.